src/modules/Hosts/HostsUILib/HostsMainPage.xaml

```diff
@@ -412,23 +412,23 @@
                 <ListView.ItemTemplate>
                     <DataTemplate x:DataType="models:Entry">
                         <Grid
+                            Margin="0"
                             AutomationProperties.Name="{x:Bind Address, Mode=OneWay}"
                             Background="Transparent"
                             ColumnSpacing="8">
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="256" />
+                                <ColumnDefinition Width="*" MinWidth="150" />
                                 <!--  Address  -->
-                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="*" MinWidth="120" />
                                 <!--  Comment  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="20" />
                                 <!--  Status  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="20" />
                                 <!--  Duplicate  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="40" />
                                 <!--  ToggleSwitch  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="54" />
                                 <!--  DeleteEntry  -->
-                                <ColumnDefinition Width="Auto" />
```

Comment:
This was an empty column which has been never used. It was the course for the misaligned buttons on small windows.